src/components/GetTodos.tsx

```diff
@@ -0,0 +1,66 @@
+import React, { useEffect, useState } from "react";
+import "./Todo.css";
+import { Todo } from "./useFetch";
+import useFetch from "./useFetch";
+
+export default function GetTodos() {
+
+  let {todos,loading,error}=useFetch('http://localhost:8000/todos')
+  const [title, setTitle] = useState<string>("");
+  const [todolist, setTodolist] = useState<Todo[]>([]);
+  const [nextId, setNextId] = useState<number>(1);
+
+  useEffect(()=>{
+    setTodolist(todos)
+  },[todos])
+
+  const handleAdd = () => {
+    setTodolist([...todolist, { id: nextId, title: title, isCompleted: false }]);
```

Comment:
If you get 5 todos from api and you are adding new one, then what will the id for it ?

Reply:
Done